proposals/0001-keywords-as-argument-labels.md

```diff
@@ -1,8 +1,8 @@
 # Allow (most) keywords as argument labels
 
-* Proposal: [SE-0001](https://github.com/apple/swift-evolution/blob/master/proposals/0001-keywords-as-argument-labels.md)
+* Proposal: [SE-0001](0001-keywords-as-argument-labels.md)
```

Comment:
@jessesquires Most of the recent proposals have those links relative to the current branch.